react/Header/Navigation/Navigation.less

```diff
@@ -64,10 +64,13 @@
   margin-left: 5px;
   @media only screen and (max-width: @container-width) {
     position: absolute;
-    top: -4px;
-    right: (@gutter-width / 2);
+    right: -(@gutter-width / 4);
+    top: -3.5 * @row-height;
```

Comment:
Again not certain this is a multiple of row height. We should take care to use these principals only where it makes sense, not for all x and y values.